NppJSONViewer/NppJsonViewer/JsonViewDlg.cpp

```diff
@@ -164,11 +164,13 @@ void JsonViewDlg::DrawJsonTree()
     HTREEITEM rootNode = nullptr;
     rootNode           = m_hTreeView->InitTree();
 
+    const bool isJsonFile = m_Editor->IsJsonFile();
     const std::string txtForParsing = m_Editor->GetJsonText();
 
-    if (txtForParsing.empty())
+    if (!isJsonFile || txtForParsing.empty())
```

Comment:
This creates problem. 
1. Open a new empty document
2. Paste some json text e.g. `{"object": "ObjectName"}`. Please note, when opening new document, document type is "Normal text file"
3. Open JsonViewer panel.
4. It does not show json tree.

This is a valid usecase. Many times, we copy json text from some other source like like file, http response etc. and paste in new document to visualize and find errors.

Reply:
right, fixed it